### PR TITLE
changing elements styling

### PR DESCRIPTION
**Pull request by bishoymelek**
Changing styling for H1,H2,H3, P .. as per WePublish current editor.

### CODE REVIEW

**Review by penk85 — APPROVED** [review]

**Review by camilleory — APPROVED** [review]

File: packages/core/src/DreifussWysiwygEditor.tsx [hunk 1]
```diff
@@ -171,124 +172,126 @@ function DreifussEditor(props: DreifussWysiwygEditorOptions) {
...
-      options={options}
-      editableProps={editableProps as EditableProps}
-      initialValue={JSON.parse(
-        JSON.stringify(value?.map(block => ({...block, id: Math.random()})))
-      )}>
-      <ToolbarBalloon editor={editorRef} />
-      {!displayOnly && (
-        <HeadingToolbar>
-          {enablePlugins.basicElements && <ToolbarBasicElementsButtons editor={editorRef} />}
+    <div className="dreifuss-wrapper">
```
Comment (roosnic1) [inline packages/core/src/DreifussWysiwygEditor.tsx:175]: what is the reason to put `<Plate>` inside the div

**Review by roosnic1 — COMMENTED** [review]
just a question

File: packages/core/src/DreifussWysiwygEditor.tsx (hunk 1 above)
Comment (bishoymelek) [reply packages/core/src/DreifussWysiwygEditor.tsx:175]: I wanted to set font-family for the plate, and thinking of moving the styling related to `displayOneLine` in the css file as well. what do you think?

File: packages/character-count-ui/src/CharacterCounter.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ const calculateCharCount = (editor?: TEditor) => {
 }
 
 export function getCharacterCount(id: string): number {
-  const editor = useStoreEditorState(id)
+  const editor = useStoreEditorRef(id)
```
Comment (bishoymelek) [inline packages/character-count-ui/src/CharacterCounter.tsx:28]: this is a hotfix for selection, now when you double click a word and try to go backwards(to left) , it's gonna work properly

**Review by roosnic1 — APPROVED** [review]

**Pull request merged**